pyanaconda/payload/dnf/payload.py

```diff
@@ -1540,8 +1558,20 @@ def update_base_repo(self, fallback=True, checkmount=True):
         # We need to check this again separately in case REPO_FILES were set above.
         if source_type in SOURCE_REPO_FILE_TYPES:
 
-            # If this is a kickstart install, just return now
-            if flags.automatedInstall:
+            # If this is a kickstart install, just return now as we normally do not
+            # want to read the on media repo files in such a case. On the other hand,
+            # the local repo files are a valid use case if the system is subscribed
+            # and the CDN is selected as the installation source.
+            if is_module_available(SUBSCRIPTION):
```

Comment:
I think that you should check the type first as well.

Reply:
OK, did that.